### PR TITLE
new FW TP format

### LINKED ISSUE #31: RawWIBTp for WIB2
Accommodating latest WIB formats ("WIB2") in the TPG firmware forces a change in the output TP header, to accomodate new bitwidths.  This means we need a new TP header for use with WIB2.

Rather than modifying RawWIBTp I suggest to create a new RawWIB2Tp (or something like that) under detdataformats/wib2, to properly decouple wib and wib2 formats.

### PR DESCRIPTION
**Pull request by jimbrooke**
This PR updates the FW TP format for changes that come with adoption of standard DAQ header in firmware.

This also decouples the FW TP from WIB formats, moving it to fwtp::RawTp.

### CODE REVIEW

**Comment by jimbrooke** [comment]
Note that RawWIBTp remains in the detdataformats::wib namespace, but should eventually be removed.

**Comment by hristovaivana** [comment]
This branch was tested, including raw recording, using a fake card reader, and it does not break the readout, therefore it will be merged. In the next days this branch will be used in validation of WIB2 captures and tests with patterns.

**Review by hristovaivana — APPROVED** [review]
- does not break the rest of readout
- validation tests on real data and patterns to be done

**Review by hristovaivana — APPROVED** [review]

File: include/detdataformats/fwtp/RawTp.hpp [hunk 1]
```diff
@@ -0,0 +1,248 @@
...
+  o << "Printing raw TP frame:" << '\n';
+  o << rwtp.m_head << '\n';
+  return o;
+}
+
+} // namespace wib
+} // namespace detdataformats
+} // namespace dunedaq
+
+#endif // DETDATAFORMATS_INCLUDE_DETDATAFORMATS_FWTP_RAWTP_HPP_
```
Comment (hristovaivana) [inline include/detdataformats/fwtp/RawTp.hpp:248]: Format is ok.

**Review by hristovaivana — APPROVED** [review]
I try to Approve.

**Review by roland-sipos — APPROVED** [review]
LGTM

**Pull request merged**